nearcore/src/migrations.rs

```diff
@@ -478,6 +479,28 @@ pub fn migrate_24_to_25(path: &Path) {
     set_store_version(&store, 25);
 }
 
+/// Fix an issue with block ordinal (#5761)
+pub fn migrate_30_to_31(path: &Path, near_config: &NearConfig) {
```

Comment:
Maybe let's add the comment with "expected amount of time" such migration is going to take ?

Reply:
This is difficult because it is a moving target. The time it took to migrate the node I used to test is not necessarily reflective of the time it will take to migrate a node when this change is released. Still, I added a note to document the amount of time it takes now.

Reply:
Is there a way to prevent migration if I am using https://github.com/aurora-is-near/partner-relayer-deploy/  and starting a new node from scratch ?